odooconnector_base/unit/backend_adapter.py

```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
+# © 2015 Malte Jacobi (maljac @ github)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import logging
+
+import oerplib.error
```

Comment:
You mean like trying to import and catch a import error for the case that the external dependencie (oerplib) is not installed?

```python

try:
    import oerplib.error
except ImportError:
    pass   
```